swagger.json

```diff
@@ -3119,6 +3196,215 @@
                     }
                 }
             }
+        },
+        "Asset": {
+            "type": "object",
+            "description": "Basic information of an asset",
+            "required": [
+                "id"
+            ],
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "Asset ID",
+                    "format": "int64",
+                    "example": 1
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Asset name",
+                    "example": "Trailer 123"
+                },
+                "cable": {
+                    "type": "array",
+                    "description": "The cable connected to the asset",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "assetType": {
+                                "type": "string",
+                                "description": "Asset type",
+                                "example": "Reefer (Thermo King)"
```

Comment:
should we be giving more examples? I think it might make sense for us internally but may be more difficult for a user to understand what are the different types of assets

Reply:
i don't think the docs allow for multiple examples, but if there's confusion from the users we can look into explaining the options further in the description